.github/workflows/system_tests.yaml

```diff
@@ -24,6 +24,7 @@ jobs:
           RCC_BINARY_PATH: C:\windows64\rcc.exe
           RUN_FOR: 300
       - uses: actions/upload-artifact@v4
+        if: success() || failure()
```

Comment:
Can we use `if: always()` here?

Reply:
`if: always()` will cause this change to also trigger, if the workflow was cancelled, so does not make much sense here.